components/layout_2020/display_list/mod.rs

```diff
@@ -118,6 +118,9 @@ pub(crate) struct DisplayListBuilder<'a> {
     /// text, image, non-white canvas or SVG). Used by metrics.
     /// See https://w3c.github.io/paint-timing/#first-contentful-paint.
     is_contentful: bool,
+
+    #[cfg(debug_assertions)]
+    debug_print_indent: usize,
```

Comment:
I think it would be better to use the `PrintTree` infrastructure for this.

Reply:
oo i wasn’t aware of that, looking into it :)

Reply:
done in ff39a86828490e6f4056810850d792cc1760cd28